specs/wp-signup-spec.js

```diff
@@ -1188,18 +1543,89 @@ describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function() {
 				'The displayed URL on the view blog page is not as expected'
 			);
 		} );
+	} );
 
-		if ( locale === 'en' ) {
-			step( 'Can see the correct blog title displayed', async function() {
-				const viewBlogPage = await ViewBlogPage.Expect( driver );
-				const title = await viewBlogPage.title();
-				return assert.strictEqual(
-					title,
-					'Site Title',
-					'The expected blog title is not displaying correctly'
-				);
+	describe( 'Basic sign up for a free blog @parallel @email @canary @ie11canary', function() {
```

Comment:
Can you please remove `@canary` and `@ie11canary` from this as we only want a single canary and the non-checklist one will be good for now to ensure backwards compatibility with existing PRs